tests/shared/src/test/scala-2.13/scala/meta/tests/parsers/dotty/Scala3PositionSuite.scala

```diff
@@ -9,35 +9,47 @@ class Scala3PositionSuite extends BasePositionSuite(dialects.Scala3) {
   checkPositions[Type]("A | B")
   checkPositions[Type](
     "[X] =>> (X, X)",
-    """|Type.Bounds [X@@] =>> (X, X)
+    """|Type.ParamClause [X]
+       |Type.ParamClause [X@@] =>> (X, X)
+       |Type.Bounds [X@@] =>> (X, X)
        |Type.Tuple (X, X)
        |""".stripMargin
   )
   checkPositions[Type](
     "[_] =>> Unit",
-    """|Type.Param _
+    """|Type.ParamClause [_]
+       |Type.Param _
        |Name.Anonymous _
+       |Type.ParamClause [_@@] =>> Unit
        |Type.Bounds [_@@] =>> Unit
        |""".stripMargin
   )
-  checkPositions[Stat]("inline def f = 1")
+  checkPositions[Stat](
+    "inline def f = 1",
+    "Type.ParamClause inline def f @@= 1"
+  )
   checkPositions[Stat](
     "open trait a",
-    """|Ctor.Primary open trait a@@
+    """|Type.ParamClause open trait a@@
+       |Ctor.Primary open trait a@@
        |Template open trait a@@
        |Self open trait a@@
        |""".stripMargin
   )
 
   checkPositions[Stat](
     "extension [A, B](i: A)(using a: F[A], G[B]) def isZero = i == 0",
-    """|Type.Bounds extension [A@@, B](i: A)(using a: F[A], G[B]) def isZero = i == 0
+    """|Type.ParamClause [A, B]
+       |Type.ParamClause extension [A@@, B](i: A)(using a: F[A], G[B]) def isZero = i == 0
```

Comment:
Same here, ParamClause should contain the whole [A, B] only, no? It's also present in a number of other cases.